app/upgrades/nextupgrade/upgrades.go

```diff
@@ -29,11 +30,49 @@ func CreateUpgradeHandler(
 			return vm, fmt.Errorf("failed to migrate ICS outstanding downtime: %w", err)
 		}
 
+		recalculateSlashingMissedBlocksCounter(ctx, keepers)
+
 		ctx.Logger().Info(fmt.Sprintf("Migration {%s} applied", UpgradeName))
 		return vm, nil
 	}
 }
 
+// Sometime long ago we decreased SlashWindow to 36k, from that time MissedBlockCounter is wrong
```

Comment:
i think it's right to add a comment, that it's related to pion-1 only and it's noop operation on neutron-1

Reply:
Right